test/TPP/transform/transform-convolutions.mlir

```diff
@@ -269,7 +274,7 @@ transform.sequence failures(propagate) {
     %4 = transform.structured.map_linalg_to_tpp filter{["tpp.relu"]} in %3
 
     // Fuse relu and conv on the three outermost loops
-    %5, %loop:3 = transform.structured.fuse %4 { tile_sizes = [1, 1, 1, 0, 0] }
+    %5, %loop:5 = transform.structured.fuse %4 { tile_sizes = [1, 1, 1, 1, 1] }
```

Comment:
What's the point of this change?

Reply:
I want to fuse the elementwise op entirely.